WhaleSpotting/Repositories/WhaleSightingRepo.cs

```diff
@@ -43,7 +43,8 @@ public List<WhaleSightingResponse> ListApprovedSightings()
             .Include(ws => ws.User)
             .Include(ws => ws.WhaleSpecies)
             .Select(x => new WhaleSightingResponse(x))
-            .AsEnumerable().ToList();
+            .AsEnumerable()
```

Comment:
Thanks, can we remove the `.AsEnumerable()`?

Reply:
As explained above, removing .AsEnumerable() leads to errors.